core/controller/src/main/resources/application.conf

```diff
@@ -15,7 +15,7 @@ akka.http {
     # The controller holds connections up to 60s for blocking invokes, and
     # all other operations are expected to complete quickly. We allow a grace
     # period in addition to the blocking invoke timeout.
-    request-timeout = 90s
+    request-timeout = 65s
```

Comment:
system.basic.WskRestSequenceTests.Wsk Sequence should execute a sequence in blocking fashion and finish execution even if longer than blocking response timeout

This test case sets the payload exactly to 65000 milliseconds. I am not sure request-timeout = 65s is sufficient for this test case to complete.

Reply:
Thanks for pointing out this test. I didn't have this in mind.
I'll take care.

Reply:
I looked at the tests. I think you are referring to the test, that transforms a blocking request into a non-blocking one.
The timeout I change has nothing to do with this test. If the tests starts failing with 503 now, it would instead fail with a 200 without my changes.

Reply:
This used to be my concern. Thank you for the explanation.